### ISSUE TITLE
Install with vim-plug

### ISSUE THREAD

**Issue by ptwales** [neoclide/coc-python#250]
I noticed that [coc-metals](https://github.com/scalameta/coc-metals) can be installed with either `:CocInstall` or with `vim-plug`.  I prefer the `vim-plug` method since it can be saved to my dotfiles. I was wondering if I can do the same for `coc-python` (or any other coc plugin for that matter).

I seem to have succesfully installed `coc-python` with
```vimscript
Plug 'neoclide/coc-python', {'do': 'yarn install --frozen-lockfile'}
```
Is that the correct way to install coc-python with `vim-plug`?  If not, what is and would it be appropriate to add it to the README as an alternative installation method?

**Comment by typenil** [comment]
I like to specify plugins in `init.vim` with `let g:coc_global_extensions = ['coc-python', 'coc-css', 'coc-so-on', 'coc-so-forth']`. I like having them there more than using Plug.

**Comment by fannheyward** [comment]
@ptwales yes, it's the correct way https://github.com/neoclide/coc.nvim/wiki/Using-coc-extensions#use-vims-plugin-manager-for-coc-extension
